frontend/src/components/header/Header.vue

```diff
@@ -40,8 +41,9 @@ const props = defineProps({
             </Button>
             <span class="header-title">
                 {{ props.title }}
-                <strong class="subtitle">{{ props.subtitle }}</strong>
-                <strong>{{ environmentLabel }}</strong>
+                <strong class="subtitle"
+                    >{{ props.subtitle }} {{ environmentLabel }}</strong
+                >
```

Comment:
Fixing an issue where the screen reader speaks separately the two elements. It should speak as an one line.